### DISCUSSION TITLE
CipherOS rom uploading issue

### DISCUSSION THREAD

**Discussion by ItsAdityan** [ROM-builders/temporary#8583] [Q&A] [answered]
https://cirrus-ci.com/build/6060883813072896

**Comment by Apon77** [comment, marked as answer]
> https://cirrus-ci.com/build/6060883813072896

CipherOS-3.2-EXODUS-cipher_vince-20220410-1444-UNOFFICIAL-VANILLA.zip 
or 
CipherOS-Sv2-vince-OTA-20220410-userdebug.zip
which one need to be uploaded?

**Reply by ItsAdityan** [reply]
CipherOS-3.2-EXODUS-cipher_vince-20220410-1444-UNOFFICIAL-VANILLA.zip this one

**Reply by Apon77** [reply]
> CipherOS-3.2-EXODUS-cipher_vince-20220410-1444-UNOFFICIAL-VANILLA.zip this one

Uploaded and fixed
